### ISSUE TITLE
Design App Icon

### ISSUE THREAD

**Issue by jamietsao** [jamietsao/codepath-mapstagram#60]

**Comment by kristen** [comment]
Did we want to use this icon for sure: http://thenounproject.com/term/map-marker/37000/?
I think @mbatilando was going to modify it a little in photoshop. I was going to pick up this story and do it fast so we could see what it looked like, but realized we don't have the icon yet.

**Comment by jamietsao** [comment]
Yeah, let's go with this one.  Bonus points if we can modify it so that the pin shape looks better.

**Comment by mbatilando** [comment]
Could I get an icon with the shape we want to copy?

**Comment by jamietsao** [comment]
I signed up for NounPro, and downloaded the icon in PNG:
![noun_37000](https://cloud.githubusercontent.com/assets/253431/6771922/e2f15e60-d0ab-11e4-96c1-c24ea5544958.png)

We want PNG yeah?  Not SVG?

**Comment by mbatilando** [comment]
Yeah, PNG is what we want. Didn't we want to copy the shape of another icon and just put the mountains, sun and clouds in?

**Comment by jamietsao** [comment]
I like the shape of this one:
http://thenounproject.com/term/location/11205/

Yeah, wasn't sure which direction you were gonna make the changes.  So do you want me to download this one instead?

**Comment by mbatilando** [comment]
Yep that looks good. Are we downloading/buying now or later?

**Comment by jamietsao** [comment]
I already signed up for NounPro ($9.99 for a month of unlimited downloads - @kristen's great idea) so I downloaded the one above.  I'm gonna download the rest of the icons we agreed on tonight (have to leave for dinner soon).  Did you want me to download the other pin (http://thenounproject.com/term/location/11205/)?

**Comment by jamietsao** [comment]
Here you go if case you need it:

![noun_11205](https://cloud.githubusercontent.com/assets/253431/6771987/4f6ef4f2-d0ad-11e4-8835-3a580ddab9ee.png)

**Comment by mbatilando** [comment]
Awesome!

**Comment by kristen** [comment]
Thanks guys! @mbatilando let me know when you get a chance to change the icon and I'll make a version of the app icon (unless you want to do it)

**Comment by mbatilando** [comment]
![mapstagram-combined](https://cloud.githubusercontent.com/assets/1978001/6773123/97b3d14e-d0c9-11e4-987b-dea812bfc2e3.png)

Here you go!

**Comment by jamietsao** [comment]
Nice job!

**Comment by kristen** [comment]
Hell ya! I'll get started on this then.

**Comment by kristen** [comment]
![image](https://cloud.githubusercontent.com/assets/1939235/6774378/67c69a3a-d0dc-11e4-9e77-014490f82a9e.jpg)

**Comment by kristen** [comment]
I can try scaling the map marker bigger, but I've tried different sizes and I'm not sure it's using the app icon real estate very well. What do you guys think? I think we might want an icon that fits better in a square, this is too tall/narrow.

**Comment by kristen** [comment]
(please ignore background photo lol)

**Comment by mbatilando** [comment]
I agree, maybe scaling it a bit bigger will make it look better. How do we feel about adding a slight gradient similar to the other icons? (lol on your cute letter btw)

**Comment by jamietsao** [comment]
I actually think it looks pretty good, but scaling it a little is probably good.  

Nice job, guys!

**Comment by kristen** [comment]
@mbatilando I was wondering about the gradient as well. Some apps have it and some don't. Maybe I should let you mess with this since you are the photoshop wiz and I'm struggling with Gimp...

(super embarrassing with the letter, lol. didn't realize it until I already posted the photo)

**Comment by jamietsao** [comment]
My friend whipped up some icons:

![mapsta_app_logo](https://cloud.githubusercontent.com/assets/253431/6786801/b5741eaa-d14b-11e4-9d62-74c1174ffeb4.png)

![mapsta_app_logo_2](https://cloud.githubusercontent.com/assets/253431/6786820/c3d7ddd8-d14b-11e4-9100-9185bbd2528e.png)

![mapsta_app_logo_3](https://cloud.githubusercontent.com/assets/253431/6787002/f46d7092-d14c-11e4-8c73-1cc62c34234f.png)

I think the 2nd/3rd (same except colors are inverted) ones are pretty cool.  It's a shutter (in case it's not obvious).

Anyways, just wanted to show these in case you guys liked them.  We can discuss more tonight!

**Comment by jamietsao** [comment]
Here's what it would look like with our green:

![mapsta_icon_screenshot](https://cloud.githubusercontent.com/assets/253431/6787302/1d478758-d14f-11e4-8e7e-a5a06649f7a8.png)

**Comment by kristen** [comment]
That is sweet! Thanks to your friend for doing that. Yes, let's talk about it tonight

**Comment by mbatilando** [comment]
Looks good! Maybe a bit of a gradient would make it look better? What do you guys think?

![mapstagram-full](https://cloud.githubusercontent.com/assets/1978001/6794301/610e0866-d18d-11e4-990c-c3c6722b190f.png)
